src/resources/2017/MummingTrunk.ts

```diff
@@ -0,0 +1,32 @@
+import { Familiar, toFamiliar } from "kolmafia";
+import { NumericModifier } from "../../modifierTypes";
+import { get } from "../../property";
+
+function toModifier(input: string): NumericModifier {
+  const regExp = new RegExp(/(Experience)+ \(([^)]+)\)/);
```

Comment:
Why do you want to match `ExperienceExperienceExperience (Moxie)`? Also, just do `Experience \((.*?)\)`